### ISSUE TITLE
The Reflow Criterion from WCAG 2.1 Fails on All Pages

### ISSUE THREAD

**Issue by WayneEDick** [w3c/low-vision-a11y-tf#82]
<h1>GitHub Fails the Reflow Criterion SC of WCAG 2.1

<p>	
	- Issues pages are the worst. However, <br>
	<strong>the reflow failure occurs on all GitHub pages.</strong>  
</p>
<p>	
	My work around for Issues is the style sheet rule below:
<br>
	div.unminimized-comment.editable-comment.comment.previewable-edit.js-comment.js-task-list-container.timeline-comment.reorderable-task-lists, 
	div.timeline-comment {max-width: 20em;}
<br>
	Most people with low vision cannot write personal style sheets.	
</p>
<p>
	Personal Impact: I was excluded from most of the discussion in formulating WCAG 2.1 because dissussions were conducted in this non-reflow format. Many other participants from the Low Vision Task Force expressed the same problem.
</p>
<p>
	- Change comparison is unusable in large print. My workaround<br>
	I use my IDE for GitHub transactions. Unfortunately this doesn't work for Issues.<br>
	Peraonal impact: For me only issues. For many others with low vision facility with IDE's is less developed. 
</p>
